README.md

```diff
@@ -17,54 +19,146 @@
+    * **Easy Sharing and Storage:** MedRecords can be exported as `.ron` files (Rusty Object Notation), a human-readable format that is easy to share, store, and version control.
 
-We recommend to use `pip` to install the latest version of MedModels:
+ * **Query Engine**:
+    * **Flexible Querying:**  Efficiently query nodes and edges of the MedRecord based on various criteria, including attributes, labels, and relationships.
+    * **High-Performance Implementation:**  Leverages a Rust backend for fast and efficient query execution, even on large datasets.
 
-```
+* **Treatment Effect Analysis:**
+
+    *  **Estimating Treatment Effects:**  Provides a range of methods for estimating treatment effects from observational data, including:
+        *  **Continuous Outcomes:** Analyze treatment effects on continuous outcomes.
+        *  **Binary Outcomes:**  Estimate odds ratios, risk ratios, and other metrics for binary outcomes.
+        *  **Time-to-Event Outcomes:**  Perform survival analysis and estimate hazard ratios for time-to-event outcomes.
+        *  **Effect Size Metrics:** Calculate standardized effect size metrics like Cohen's d and Hedges' g.
+
+    * **Matching:**
+        * **(High Dimensional) Propensity Score Matching:**  Reduce confounding bias by matching treated and untreated individuals based on their propensity scores.
+        * **Nearest Neighbor Matching:**  Match individuals based on similarity in their observed characteristics.
+
+## Getting Started
+
+**Installation:**
+
+MedModels can be installed from [PyPI](https://pypi.org/project/medmodels/) using the `pip` command:
+
+```bash
 pip install medmodels
 ```
 
-For detailed information on how to use MedModels, please read the [MedModels documentation](https://www.medmodels.de/api-reference/latest).
+**Quick Start:**
+
+Here's a quick start guide showing an example of how to use MedModels to create a `MedRecord` object, add nodes and edges, and perform basic operations.
+
+```python
+import pandas as pd
+import medmodels as mm
+
+# Patients DataFrame (Nodes)
+patients = pd.DataFrame(
+    [
+        ["Patient 01", 72, "M", "USA"],
+        ["Patient 02", 74, "M", "USA"],
+        ["Patient 03", 64, "F", "GER"],
+    ],
+    columns=["ID", "Age", "Sex", "Loc"],
+)
+
+# Medications DataFrame (Nodes)
+medications = pd.DataFrame(
+    [["Med 01", "Insulin"], ["Med 02", "Warfarin"]], columns=["ID", "Name"]
+)
+
+# Patients-Medication Relation (Edges)
+patient_medication = pd.DataFrame(
+    [
+        ["Patient 02", "Med 01", pd.Timestamp("20200607")],
+        ["Patient 02", "Med 02", pd.Timestamp("20180202")],
+        ["Patient 03", "Med 02", pd.Timestamp("20190302")],
+    ],
+    columns=["Pat_ID", "Med_ID", "Date"],
+)
+
+# Create a MedRecord object using the builder pattern
+record = mm.MedRecord.builder().add_nodes((patients, "ID"), group="Patients").build()
```

Comment:
I didn't mean you should completely remove the other stuff, just add it to the builder :D
Now it wouldn't make sense why the `medications` and `patient_medication` dataframes exist.

I meant:
```suggestion
record = (
    mm.MedRecord.builder()
    .add_nodes((patients, "ID"), group="Patients")
    .add_nodes((medications, "ID"), group="Medications")
    .add_edges((patient_medication, "Pat_ID", "Med_ID"))
    .add_group("US-Patients", nodes=["Patient 01", "Patient 02"])
    .build()
)
```